docs/api/api_data.js

```diff
@@ -1,3669 +1,3669 @@
 define({
-  api: [

```

Comment:
Alright something's up here. Idk what exactly, but this change shouldn't be 7330 lines. What's the diff? Can you look into this please?